src/actions/collection/links/refresh.js

```diff
@@ -0,0 +1,19 @@
+import collectionLinksError from './error';
+import collectionLinksPush from './push';
+
+export default function refresh(id) {
+  return async dispatch => {
+    try {
+      const resp = await fetch(`${process.env.BACKSTROKE_SERVER}/v1/links/${id}`)
```

Comment:
Here's an example of how to pass along the proper cookie to get auth to work:

```javascript
const resp = await fetch(`${API_URL}/v1/links`, {credentials: 'include'}); // Important part is {credentials: 'include'}
```

From https://github.com/backstrokeapp/dashboard/blob/master/src/actions/collection/links/fetch.js#L10

Also, you'll want to use the host found in the `API_URL` constant (instead of the `BACKSTROKE_SERVER` environment variable). An example of its usage can be found here: https://github.com/backstrokeapp/dashboard/blob/master/src/actions/collection/links/fetch.js#L3